### ISSUE TITLE
library_call_linter() doesn't allow suppressed library() calls

### ISSUE THREAD

**Issue by MichaelChirico** [r-lib/lintr#2163]
```r
lint("
suppressMessages({
  library(a)
  library(b)
})
",
library_call_linter())
# <text>:3:3: warning: [library_call_linter] Move all library calls to the top of the script.
#   library(a)
#   ^~~~~~~~~~
# <text>:4:3: warning: [library_call_linter] Move all library calls to the top of the script.
#   library(b)
#   ^~~~~~~~~~
```
